zebrad/tests/acceptance.rs

```diff
@@ -12,15 +12,24 @@ use toml;
 use zebra_test::prelude::*;
 use zebrad::config::ZebradConfig;
 
-pub fn tempdir(create_config: bool) -> Result<(PathBuf, impl Drop)> {
-    let dir = TempDir::new("zebrad_tests")?;
+enum ConfigMode {
+    NoConfig,
+    Ephemeral,
+    Persistent,
```

Comment:
@teor2345 Does this approach to test the persistent database look okay to you? I modified an existing test currently. It passes locally. Should I leave it as is or make a new test?

Reply:
I like the enum, it's easier to understand than a bool.

I think it's ok to modify the existing test. We expect `zebrad start` to use a sled database.

Reply:
Actually, I think it might be better to have separate `cache_dir` and `ephemeral` tests.

If we have separate tests, it will be easier to diagnose test failures. And it will be easier to maintain our tests - we want to keep testing these features, even if we add or modify tests. (If we test too many things in the one test function, its easy to accidentally delete one of them.)

Here's what the ephemeral test would look like:

> As part of the acceptance tests for this feature, we should write a test which makes sure that cache_dir is ignored when ephemeral is true. This test should set cache_dir to a directory inside a new temp dir, then check that zebrad does not use the directory for its state.